Charon/filetypes/GCodeSocket.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# libCharon is released under the terms of the LGPLv3 or higher.
+#
+# This class is used to read GCode stream that are served
```

Comment:
This documentation should be in the class (so in between class and init), that way it's actually set as the documentation.